configs/default.yaml

```diff
@@ -30,6 +31,11 @@ rules:
   eqeqeq: 2
   func-names: 1
   guard-for-in: 2
+  import/order:
+    - error
+    - groups: [[builtin, external, internal], [parent, sibling, index], [unknown]]
+      newlines-between: always
+      alphabetize: {order: asc}
```

Comment:
While this is correct, IMHO using yaml all the way down produces more readable config:

```
  import/order:
    - error
    - groups:
      - - builtin
        - external
        - internal
      - - parent
        - sibling
        - index
      - - unknown
      newlines-between: always
      alphabetize:
        order: asc
```

Reply:
When I was writing this PR i wanted to stick with the current style - use brackets and stay in one line as much as possible. At the same time I personally would opt for version you proposed. 
As long as this change doesn't affect essence of this PR and it's only semantic issue, we can make an appointment that if no one objects your proposal to the end of the vote I will implement it here.